kars/NNNN-kar-template/README.md

```diff
@@ -0,0 +1,124 @@
+<!--
+**Note:** When your KAR is complete, all of these comment blocks should be removed.
+
+Follow the guidelines of the [documentation style guide].
+In particular, wrap lines to a reasonable length, to make it
+easier for reviewers to cite specific portions, and to minimize diff churn on
+updates.
+
+[documentation style guide]: https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md
+
+To get started with this template:
+
+- [ ] **Create an issue in kubernetes-sigs/wg-ai-conformance**
+  When filing an AI conformance requirement tracking issue, please make sure to complete all
+  fields in that template. One of the fields asks for a link to the KAR. You
```

Comment:
There should be an accompanying issue template for this, right? Will that be part of this PR?